Bio/SearchIO/InfernalIO/infernal_text.py

```diff
@@ -0,0 +1,543 @@
+# Copyright 2024 by Samuel Prince. All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+
+"""Bio.SearchIO parser for Infernal plain text output format."""
+
+import re
+import operator
```

Comment:
Style suggestion (feel free to ignore):

```suggestion
import operator
import re
```